flagsmith/flagsmith.py

```diff
@@ -124,20 +140,54 @@ def __init__(
                         "in the environment settings page."
                     )
 
-                self.environment_data_polling_manager_thread = (
-                    EnvironmentDataPollingManager(
+                if self.use_stream:
```

Comment:
Can we add a check to make sure streams are only enabled when running in local evaluation mode(throw an error if that's not the case)